_build/config.json

```diff
@@ -93,6 +96,12 @@
         },{
             "key": "skin",
             "value": "modx"
+        },{
+            "key": "tinymcerte.extendend_valid_elements",
+            "value": "\"span[class] i[class]\""
+        },{
+            "key": "tinymcerte.content_css",
```

Comment:
Good day!

We may update this PR with integration with NPM or upgrade font-awesome?

Reply:
@carnevlu Last version: v5.8.0 https://fontawesome.com/start

Please update your PR. Thank you!